marklogic-data-hub-central/ui/e2e/cypress/support/pages/browse.tsx

```diff
@@ -144,8 +144,13 @@ class BrowsePage {
     return cy.get('[data-cy=selected-facet-block]');
   }
 
-  getSelectedFacet(facet: string) {
-    return cy.get('#selected-facets > button').contains(facet);
+  computeStartDateOfTheWeek() {
```

Comment:
removed getSelectedFacet to reuse existing locators. getSelectedFacet didnt work for null check of an element since it was using contains to match target element.